e2e/kube.go

```diff
@@ -76,19 +76,24 @@ func getClusterKubeconfigBytes(ctx context.Context, resourceGroupName, clusterNa
 }
 
 // this is a bit ugly, but we don't want to execute this piece concurrently with other tests
-func ensureDebugDaemonsets(ctx context.Context, kube *Kubeclient) error {
-	hostDS := getDebugDaemonsetTemplate(hostNetworkDebugAppLabel, "nodepool1", true)
+func ensureDebugDaemonsets(ctx context.Context, kube *Kubeclient, isAirgap bool) error {
+	hostDS := getDebugDaemonsetTemplate(hostNetworkDebugAppLabel, "nodepool1", true, isAirgap)
 	if err := createDebugDaemonset(ctx, kube, hostDS); err != nil {
 		return err
 	}
-	nonHostDS := getDebugDaemonsetTemplate(podNetworkDebugAppLabel, "nodepool2", false)
+	nonHostDS := getDebugDaemonsetTemplate(podNetworkDebugAppLabel, "nodepool2", false, isAirgap)
 	if err := createDebugDaemonset(ctx, kube, nonHostDS); err != nil {
 		return err
 	}
 	return nil
 }
 
-func getDebugDaemonsetTemplate(deploymentName, targetNodeLabel string, isHostNetwork bool) string {
+func getDebugDaemonsetTemplate(deploymentName, targetNodeLabel string, isHostNetwork, isAirgap bool) string {
```

Comment:
I forget, do we create separate e2e clusters just for airgap scenarios?

im just wondeirng how this would play with other scenarios since every scenario ends up using this

Reply:
Yep we do create separate clusters for e2e. It should still use the mcr path for non air gap tests

Reply:
If they were the same clusters, they would be subjugated to the airgap rules